packages/next-on-pages/templates/_worker.js/index.ts

```diff
@@ -12,25 +15,29 @@ declare const __ENV_ALS_PROMISE__: Promise<null | AsyncLocalStorage<unknown>>;
 
 export default {
 	async fetch(request, env, ctx) {
+		patchFetchToAllowBundledAssets();
 		const envAsyncLocalStorage = await __ENV_ALS_PROMISE__;
 		if (!envAsyncLocalStorage) {
 			return new Response(
 				`Error: Could not access built-in Node.js modules. Please make sure that your Cloudflare Pages project has the 'nodejs_compat' compatibility flag set.`,
 				{ status: 503 }
 			);
 		}
-		return envAsyncLocalStorage.run({ ...env, NODE_ENV: __NODE_ENV__ }, () => {
-			const adjustedRequest = adjustRequestForVercel(request);
+		return envAsyncLocalStorage.run(
+			{ ...env, NODE_ENV: __NODE_ENV__ },
+			async () => {
```

Comment:
Does this now need to be async?

Reply:
in https://github.com/cloudflare/next-on-pages/pull/317 I've added some eslint Promises related rules, one of those require you to make async functions that return a promise: [promise-function-async](
https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/promise-function-async.md)

As I touched this file vscode complained about it so I added the `async`, I think that our eslint config might not be picking up stuff in the `templates` directory, I'll address that in a separate PR

For the added `async` and `promise-function-async` I guess it's kinda nice that we get the extra indication that functions are asynchronous(-like) instead of having to read their content, but I am lukewarm about it, if you prefer I am totally ok in removing these.